src/assets/scss/base/_global.scss

```diff
@@ -0,0 +1,5 @@
+$font-size-default: 16px;
```

Comment:
Just move this to settings and you're good to merge. Make sure settings is imported before global in main.scss